system/core/pages.php

```diff
@@ -534,3 +567,60 @@ function image_resize($src, $dst, $width, $height) {
 		}
 		return true;
 	}
+
+	function getButtonInfo($_args) {
+		/* 0 = buttonName
+		*  1 = buttonName
+		*  2 = language (or null)
+		*/
+		Global $O_O;
+		$_button = "";
+		$_language = "";
+
+		if (sizeof($_args) > 1) {$_button = $_args[1];} else {$_button = "undifined";}
+		$a = array("%20","%22");
+		$b = array(" "," ");
+	  $_button = str_replace($a,$b,$_button);
```

Comment:
What does this replacement do? Replace escaped spaces (%20) with doublequotes (%22)? And one kind of spaces with another? This code can really use a comment to explain what it is doing and why.